src/app/components/event/eventController.js

```diff
@@ -9,9 +9,12 @@
 import './eventStyles.styl'
 
 angular.module('app')
-  .controller('EventController', function ($log) {
+  .controller('EventController', function ($log, $scope) {
     var vm = this
     vm.test = 'testing'
+
+    $scope.eventDescr=['Event1', 'Event2', 'Event3']
```

Comment:
Eslint is saying:
![screen shot 2017-02-22 at 6 30 01 pm](https://cloud.githubusercontent.com/assets/13861069/23237789/f79ea290-f92c-11e6-9d2e-87d80d88121f.png)